core/integration/module_test.go

```diff
@@ -359,14 +359,19 @@ func TestModuleGit(t *testing.T) {
 
 			if tc.sdk == "go" {
 				logGen(ctx, t, modGen.Directory("."))
+				out, err := modGen.
+					With(daggerQuery(`{bare{containerEcho(stringArg:"hello"){stdout}}}`)).
+					Stdout(ctx)
+				require.NoError(t, err)
+				require.JSONEq(t, `{"bare":{"containerEcho":{"stdout":"hello\n"}}}`, out)
+			} else {
+				out, err := modGen.
+					With(daggerQuery(`{bare{myFunction(stringArg:"hello"){stdout}}}`)).
+					Stdout(ctx)
+				require.NoError(t, err)
+				require.JSONEq(t, `{"bare":{"myFunction":{"stdout":"hello\n"}}}`, out)
```

Comment:
I'll update the Python default to match Go. I did that added the tests to have unified test case but it's also nice for the user to compare the different implementations of the same thing.